geps/gep-2907/index.md

```diff
@@ -0,0 +1,109 @@
+# GEP-2907: TLS Configuration Placement and Terminology
+
+* Issue: [#2907](https://github.com/kubernetes-sigs/gateway-api/issues/2907)
+* Status: Memorandum
+
+## TLDR
+This GEP aims to define high level TLS terminology and structure within Gateway
+API to ensure that all our independent proposals related to TLS result in a
+coherent set of APIs.
+
+## Goals
+* Define high-level terminology for how we refer to TLS in Gateway API.
+* Define top level fields where TLS configuration can live.
+
+## Non-Goals
+* Add or change fields directly. (This may inspire changes in other GEPs
+  though).
+* Commit to including specific parts of TLS configuration in Gateway API. (This
+  is merely to provide space for future configuration, not a commitment that we
+  will add it to the API.)
+
+## Introduction
+
+There are essentially 4 different "buckets" that could be relevant to TLS
+configuration in Gateway API:
+
+```mermaid
+flowchart LR
+    Frontend -.->|1. Client Cert| Gateway
+    Gateway -.->|2. Server Cert| Frontend
+    Gateway -.->|3. Client Cert| Backend
+    Backend -.->|4. Server Cert| Gateway
+```
+
+### Where it can fit
+We have three different places where this configuration could live:
+
+#### A. Gateways
+We can add config that applies to an entire Gateway. Config attached at this
+layer will apply to everything behind a single VIP.
```

Comment:
Good call, this was confusing. Reworked this a bit since I'm mostly trying to describe the concept of a Gateway when it comes to TLS to someone that might not be that familiar with the rest of the API.